Lombiq.Tests.UI/Services/UITestContext.cs

```diff
@@ -89,6 +89,13 @@ public class UITestContext
     /// </summary>
     public string TenantName { get; set; } = "Default";
 
+    /// <summary>
+    /// Gets or sets the current admin prefix if a custom one is needed. When creating UI tests on a site that
+    /// has custom admin prefix this value should be set in the UI test. After that navigation methods will be able
+    /// to catch up to the custom url.
+    /// </summary>
+    public string CustomAdminUrl { get; set; }
```

Comment:
This should be used everywhere where currently it's "/Admin". See:

- `AdminMonkeyTestingUrlFilter`
- `IsOrchardPage`
- `TestAdminAsMonkeyRecursivelyAsync`

Reply:
At `AdminMonkeyTestingUrlFilter` I can't find a viable way to inject the AdminPrefix as it is not using UITestContext directly.
So next to the /admin there should be the context.AdminPrefix. Is UITestContext injectable btw?

Reply:
Don't you see `UITestContext` in `AdminMonkeyTestingUrlFilter`? ;) Please put more effort into figuring out these things on your own. You can also try injecting `UITestContext` and see if it works.

Reply:
I am putting all my effort into my current issues. I am asking because this is an Open Source Library and I don't want to spaghetti it. There is `UITestContext ` in `AdminMonkeyTestingUrlFilter` but not in the class itself just the class's method takes it as a parameter.  Which is for me probably useless because I need to supply the new instance of `StartsWithMonkeyTestingUrlFilter` with the `context.AdminPrefix` as it is a readonly property I can't change it after creating it. If I create a CTOR `AdminMonkeyTestingUrlFilter` requiring UITestContext could be a possibility but that will result in chain so I will also have to address `NotAdminMonkeyTestingUrlFilter` with the same CTOR which seems overkill and spaghetti for me.

Reply:
Well I guess I found a solution

Reply:
You see! `UITestContext` not being there in the ctor of `AdminMonkeyTestingUrlFilter` shouldn't be a problem, of course.

Reply:
I just thought it is not as clean as it should be.